test/kfp-functional-test/requirements.in

```diff
@@ -0,0 +1 @@
+kfp==1.1.2
```

Comment:
Including the script you used to update requirements as a README in this folder, make it easily findable for the next person coming to take a look.

Reply:
done